__tests__/app.test.js

```diff
@@ -26,13 +26,50 @@ describe("app/topics", () => {
       .get("/api/topics")
       .expect(200)
       .then(({ body }) => {
+        expect(body).toHaveLength(3);
         body.forEach((object) => {
           expect(object).toMatchObject({
             slug: expect.any(String),
             description: expect.any(String),
           });
-          console.log(object);
         });
       });
   });
+  describe("app/articles", () => {
+    test("200: returns an array of article objects", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body).toBeInstanceOf(Array);
+        });
+    });
+    test("200: returns an array of article objects where each object has all the correct properties", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          body.forEach((article) => {
```

Comment:
Don't forget to check the length of the array every time for those you are iterating over